src/service/axiosInstance.jsx

```diff
@@ -1,7 +1,28 @@
 import axios from 'axios';
+import { getSessionStorage } from '../utils/sessionStorage';
+import { UNAUTHENTICATED_ROUTES, BASE_URL } from '../constants/constants';
 
 const axiosInstance = axios.create({
-  baseURL: 'https://36b32v1d09.execute-api.sa-east-1.amazonaws.com/',
+  baseURL: BASE_URL,
+});
+
+axiosInstance.interceptors.request.use((req) => {
+  const token = getSessionStorage('token');
+
+  const isUnauthenticatedRoute = UNAUTHENTICATED_ROUTES.includes(req.url);
+
+  if (isUnauthenticatedRoute) {
+    return req;
+  }
+
+  if (token) {
+    req.headers = {
+      Authorization: `Bearer ${token}`,
+    };
```

Comment:
Aqui os headers da requisição estão sendo sobrescritos, certo? Se estiver sendo feita uma requisição que já contenha headers (como paginação por exemplo), ela será perdida. 

Uma maneira de preservar a configuração anterior é atualizando os headers da requisição ao invés de sobrescrevê-los:

```suggestion
    const { headers } = req
    req.headers = {
      ...headers,
      Authorization: `Bearer ${token}`,
    };
```

Reply:
Realmente Lucas, não tinha pensado nisso. Vou corrigir